### DISCUSSION TITLE
Osu (Lazer) tablet rotation doesnt take the full screen

### DISCUSSION THREAD

**Discussion by SevyfS** [ppy/osu#26394] [Q&A]
I used to rotate my tablet area of 90° using the driver (xp-pen tablet) and everything worked just fine without any issues. Osu!Lazer doesnt take the custom rotation from my driver so i had to change it manually in the settings of the game but when i do so my tablet area cant take the full screen and i dont know the reason behind it. Is there any way to fix that ?

Image of the problem: https://prnt.sc/xjAK2eMDih8I

**Comment by peppy** [comment]
You can just disable the in-game tablet support if you'd prefer to use your external settings.
